docs/source/development.rst

```diff
@@ -338,7 +349,7 @@ Linux/Mac
     export "DJANGO_SETTINGS_MODULE=hawc.main.settings.unittest"
 
     # load existing test
-    createdb hawc-fixture-test
+    createdb hawc-fixture
     manage.py load_test_db
 
     # now make edits to the database using the GUI or via command line
```

Comment:
may need to document that we need to specify the hawc-fixture database name in our settings, as documented in the local.example.py

Reply:
Sorry, this comment confused myself too, please ignore :)